src/Models/cards.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose from 'mongoose';
+
+const cardSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    default: '',
+  },
+  ownedBy: {
+    type: String,
+    required: true,
+  },
+});
+
+const Cards = mongoose.model('card', cardSchema);
+
+export default Cards;
```

Comment:
About the convention, `Model` should be singular. `Card` instead of `Cards`

https://www.coding-guidelines.lftechnology.com/docs/java/classes